fn_grpc_interface/fn_grpc_interface/util/grpc_helper.py

```diff
@@ -0,0 +1,165 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+# -*- coding: utf-8 -*-
+import os
+import sys
+import six
+import logging
+import inspect
+import grpc
+
+try:
+    # PY3
+    import importlib.util
+except ImportError as e:
+    # PY2
+    logging.debug("The Import Error raised for the module importlib trying with imp module : {}".format(e))
+    import imp
+
+
+class GrpcHelperClass(object):
+    def __init__(self, logger_object):
+        self.log_object = logger_object
+
+    def get_grpc_interface_module(self, interface_files=[], base_dir=None, files_dir=None):
+        """
+        component that imports all the modules in the current workspace and returns the given imported modules objects.
+        :param interface_files: List of all buffer pb2 files absolute path to import in the current work space
+        :param base_dir:  interface pb2 files parent directory.
+        :param files_dir: interface pb2 files directory i.e same as package name.
+        :return: returns imported interface pb2 files module objects on the successful execution, else returns None object.
+        """
+        if not isinstance(interface_files, list):
+            raise ValueError("This Parameter "'"interface_files"'" should be list data type")
+        if base_dir is None:
+            raise ValueError("The param base_dir should contain parents parent directory of gRPC interface files")
```

Comment:
Change to: "'base_dir' is not defined. 'base_dir' is the path to your interface_files directory"

Reply:
yes